components/organisms/ContinueRegistrationModalContent.tsx

```diff
@@ -0,0 +1,152 @@
+import { RegistrationStep } from "@/lib/name-registration/constants";
+import { LocalRegistrationNameData } from "@/lib/name-registration/types";
+import { useNameRegistration } from "@/lib/name-registration/useNameRegistration";
+import { BasicInfoKey, SocialAccountsKeys } from "../02-molecules";
+import { endNameRegistrationPreviouslyOpen } from "@/lib/name-registration/localStorage";
+import { useAccount } from "wagmi";
+import { Dispatch, SetStateAction } from "react";
+import { Button } from "@ensdomains/thorin";
+
+export const ContinueRegistrationModalContent = ({
+  name,
+  localNameRegistrationData,
+  setIsModalContinueRegistrationOpen,
+}: {
+  name: string;
+  localNameRegistrationData: LocalRegistrationNameData;
```

Comment:
Please make these names standardized, we have both `localNameRegistrationData` and `LocalRegistrationNameData` names being used, switching words sorting